forge-gui/res/quest/precons/Enhanced Evolution.dck

```diff
@@ -0,0 +1,99 @@
+[metadata]
+Name=Enhanced Evolution
+[shop]
+WinsToUnlock=0
+Credits=5000
+MinDifficulty=0
+MaxDifficulty=5
+[metadata]
+Description=Overpower opponents with brand-new mechanic from Ikoria: Lair of Behemoths—mutate!
```

Comment:
`with brand-new mechanic` -> `with a brand-new mechanic` or `with the brand-new mechanic` ??